lib/matplotlib/axes/_axes.py

```diff
@@ -4478,6 +4478,20 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             The marker size in points**2 (typographic points are 1/72 in.).
             Default is ``rcParams['lines.markersize'] ** 2``.
 
+            The linewidth and edgecolor can visually interact with the marker
+            size.
+
+            If the linewidth is greater than 0 and the edegecolor is anything
+            but *'none'*, then the effective size of the marker will be
+            increased by half the linewidth as the stroke drawn for the edge
+            will be centered on the edge of the shape.
+
+            If the marker size is smaller than the linewidth there will be
+            visual artifacts.
```

Comment:
Wordsmithing - take or leave as you please:

```suggestion
            The linewidth and edgecolor can visually interact with the marker
            size, and can lead to artifacts if the marker size is smaller than the linewidth. 

            If the linewidth is greater than 0 and the edegecolor is anything
            but *'none'*, then the effective size of the marker will be
            increased by half the linewidth because the stroke will be centered 
            on the edge of the shape.

```

Reply:
That is better, committed!